torch/_inductor/fx_passes/joint_graph.py

```diff
@@ -216,30 +216,34 @@ def joint_graph_passes(graph: torch.fx.GraphModule):
     return graph
 
 
-@register_graph_pattern(
-    CallFunction(
-        torch.ops.prims.convert_element_type.default,
+if not config.freezing:
```

Comment:
Can we make the `extra_check` function `not config.freezing` ? we want the check to happen at runtime, and for this to always get registerec